Description: This patch removes special handling code for Desktop folder
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 xdg-user-dirs (0.14-0ubuntu2) precise; urgency=low
 .
   * Upload translations from launchpad (lp: #549088)
Author: Sergey "Shnatsel" Davidoff <sergey@elementaryos.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- xdg-user-dirs-0.14.orig/user-dirs.defaults
+++ xdg-user-dirs-0.14/user-dirs.defaults
@@ -2,7 +2,6 @@
 #
 # The values are relative pathnames from the home directory and
 # will be translated on a per-path-element basis into the users locale
-DESKTOP=Desktop
 DOWNLOAD=Downloads
 TEMPLATES=Templates
 PUBLICSHARE=Public
--- xdg-user-dirs-0.14.orig/xdg-user-dir
+++ xdg-user-dirs-0.14/xdg-user-dir
@@ -1,8 +1,4 @@
 #!/bin/sh
 
 test -f ${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs && . ${XDG_CONFIG_HOME:-~/.config}/user-dirs.dirs
-if [ "x$1" = "xDESKTOP" ]; then
-  eval echo \${XDG_${1}_DIR:-$HOME/Desktop}
-else
-  eval echo \${XDG_${1}_DIR:-$HOME}
-fi
+eval echo \${XDG_${1}_DIR:-$HOME}
--- xdg-user-dirs-0.14.orig/xdg-user-dirs-update.c
+++ xdg-user-dirs-0.14/xdg-user-dirs-update.c
@@ -22,7 +22,6 @@ typedef struct {
 } Directory;
 
 Directory backwards_compat_dirs[] = {
-  { "DESKTOP", "Desktop" },
   { "TEMPLATES", "Templates" },
   { "PUBLICSHARE", "Public" },
   { NULL}
--- xdg-user-dirs-0.14.orig/xdg-user-dir-lookup.c
+++ xdg-user-dirs-0.14/xdg-user-dir-lookup.c
@@ -201,18 +201,6 @@ xdg_user_dir_lookup (const char *type)
   if (home_dir == NULL)
     return strdup ("/tmp");
   
-  /* Special case desktop for historical compatibility */
-  if (strcmp (type, "DESKTOP") == 0)
-    {
-      user_dir = (char*) malloc (strlen (home_dir) + strlen ("/Desktop") + 1);
-      if (user_dir == NULL)
-        return NULL;
-
-      strcpy (user_dir, home_dir);
-      strcat (user_dir, "/Desktop");
-      return user_dir;
-    }
-  
   return strdup (home_dir);
 }
 
